symphony-bdk-template/symphony-bdk-template-freemarker/src/main/java/com/symphony/bdk/template/freemarker/FreeMarkerEngine.java

```diff
@@ -59,6 +60,10 @@ private static Configuration createConfiguration() {
     cfg.setLogTemplateExceptions(false);
     cfg.setWrapUncheckedExceptions(true);
     cfg.setFallbackOnNullLoopVariable(false);
+    cfg.setNumberFormat("0.######");
```

Comment:
Does it mean we drop the fraction parts over 6 digits?

Reply:
Unfortunately yes. But I just found another parameter that we could use "computer". it has the same impact as this https://freemarker.apache.org/docs/ref_builtins_number.html#ref_builtin_c . Let me replace it.